index.html

```diff
@@ -13,17 +13,45 @@
  <header class="header">
   <nav class="nav">
    <ul class="nav_list">
-    <li class="nav_item"> <a class="nav_link" href="#"> Home </a> </li>
+    <li class="nav_item"> <a class="Home" href="#"> Home </a> </li>
     
-    <li class="nav_item"> <a class="nav_link" href="#"> About </a> </li>
+    <li class="nav_item"> <a class="About" href="#"> About </a> </li>
    
-    <li class="nav_item"> <a class="nav_link" href="#"> Blog </a> </li>
+    <li class="nav_item"> <a class="Blog" href="#"> Blog </a> </li>
 
-    <li class="nav_item"> <a class="nav_link" href="#"> Contact Us </a> </li>
+    <li class="nav_item"> <a class="Contact_Us" href="#"> Contact Us </a> </li>
     </ul>
   </nav>
+
+   <img class="logo">
+    <img src="assets/logo.png"> 
+   </img>
+
+  <div class="border_bottom">
+  
+  </div>
  </header>
 
+ <main>
+ 
+  <div class="card1">
+    <img src="assets/card.png">
+  </div>
+
+  <div class="card2">
+    <img src="assets/card.png">
+  </div>
+
+  <div class="card3">
+    <img src="assets/card.png">
+  </div>

```

Comment:
Hi @DanielNcarter. You may want to use the same class name for all of your cards since they have the same styles (e.g. class="main__card").